cmd/node/config/economics.toml

```diff
@@ -1,8 +1,8 @@
 #Economics config of the node
 [GlobalSettings]
-    TotalSupply = "20000000000000000000000000000" #20BILERD
+    TotalSupply = "200000000000000000000000000000" #20BILERD
```

Comment:
You have an extra 0. What is wrote there is actually 200 bil

Reply:
yep.